modules/nw-udn-cr-ui.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/primary_networks/about-user-defined-networks.adoc
+
+:_mod-docs-content-type: PROCEDURE                                    
+[id="nw-udn-cr-ui_{context}"]                                   
+= Creating a UserDefinedNetwork CR by using the web console
+
+You can create a `UserDefinedNetwork` custom resource by using the {product-title} web console.
+
+.Prerequisites
+
+* You have access to the {product-title} web console as a user with `cluster-admin` permissions.
```

Comment:
Does core need admin permissions for plain UDN? cc @anuragthehatter and @asood-rh

Reply:
@JoeAldinger Could you elaborate on core?

Reply:
Core OCP?